doc/development.md

```diff
@@ -91,64 +84,57 @@ make run
 ### Deploy custom APIcast Operator using OLM
 
 To install this operator on an OpenShift 4.5+ cluster using OLM for
-end-to-end testing:
-
-* Perform naming changes to avoid collision with existing APIcast Operator
-  official public operators catalog entries:
-  * Edit the `bundle/manifests/apicast-operator.clusterserviceversion.yaml` file
-    and perform the following changes:
-      * Change the current value of `.metadata.name` to a different name
-        than `apicast-operator.v*`. For example to `myorg-apicast-operator.v0.0.1`
-      * Change the current value of `.spec.displayName` to a value that helps you
-        identify the catalog entry name from other operators and the official
-        APIcast operator entries. For example to `"MyOrg apicast operator"`
-      * Change the current value of `.spec.provider.Name` to a value that helps
-        you identify the catalog entry name from other operators and the official
-        APIcast operator entries. For example, to `MyOrg`
-  * Edit the `bundle.Dockerfile` file and change the value of
-    the Dockerfile label `LABEL operators.operatorframework.io.bundle.package.v1`
-    to a different value than `apicast-operator`. For example to
-    `myorg-apicast-operator`
-  * Edit the `bundle/metadata/annotations.yaml` file and change the value of
-    `.annotations.operators.operatorframework.io.bundle.package.v1` to a
-    different value than `apicast-operator`. For example to
-    `myorg-apicast-operator`. The new value should match the
-    Dockerfile label `LABEL operators.operatorframework.io.bundle.package.v1`
-    in the `bundle.Dockerfile` as explained in the point above
-
-  It is really important that all the previously shown fields are changed
-  to avoid overwriting the APIcast operator official public operator
-  catalog entry in your cluster and to avoid confusion having two equal entries
-  on it.
-
-* [Create an operator bundle image](#generate-an-operator-bundle-image) using the
-  changed contents above
-
-* [Push the operator bundle into an external container repository](#push-an-operator-bundle-into-an-external-container-repository).
-
-* Run the following command to deploy the operator in your currently configured
-  and active cluster in $HOME/.kube/config:
-  ```sh
-  operator-sdk run bundle --namespace <mynamespace> <BUNDLE_IMAGE_URL>
-  ```
-
-  Additionally, a specific kubeconfig file with a desired Kubernetes
-  configuration can be provided too:
-  ```sh
-  operator-sdk run bundle --namespace <mynamespace> --kubeconfig <path> <BUNDLE_IMAGE_URL>
-  ```
+end-to-end testing, there are some changes needed to avoid collision with existing
```

Comment:
Do we want to still keep this information about the needed changes? It is now automatically done.
If we want to keep it for information purposes to the users then this need some rewording because as of now it is written as imperative instructions ("Edit the ....")

Reply:
removed